test/integration/acceptance/acceptance_fixture.hpp

```diff
@@ -45,6 +46,9 @@ namespace {
 #define CHECK_COMMITTED BASE_CHECK_RESPONSE(CommittedTxResponse)
 
 #define CHECK_MST_PENDING BASE_CHECK_RESPONSE(MstPendingResponse)
+
+#define CHECK_TXS(i) \
```

Comment:
Maybe name it `CHECK_TXS_NUMBER` to be more specific?

Reply:
You are right. That was an option but on the other side everyone did it even shorter, just `check`.

Reply:
Some minor issue. Add a doc string for the macro.